site/package.json

```diff
@@ -13,11 +13,12 @@
     "deploy": "yarn mosaic deploy --out snapshots --name latest --config mosaic.config.js",
     "dev": "next dev",
     "start": "next start",
-    "gen:snapshot": "yarn mosaic build --out snapshots --name latest --config mosaic.config.js && yarn copy:sitemap",
+    "gen:snapshot": "yarn mosaic build --out snapshots --name latest --config mosaic.config.js && yarn copy:sitemap && yarn gen:props",
```

Comment:
Is it correct to run `gen:props` _after_ the Mosaic site build? Shouldn't the props JSON files be generated/updated _before_ we build the site?